ui/js/values.js

```diff
@@ -32,6 +32,8 @@ treeherder.value("thPlatformNameMap", {
     "android-4-4": "Android 4.4",
     "android-4-4-armv7-api11": "Android 4.4 API11+",
     "android-5-0-armv7-api11": "Android 5.0 API11+",
+    "b2gdroid-4-0-armv7-api11": "B2GDroid 4.0 API11+",
+    "android-4-0-armv7-api11-partner1": "Android 4.0 API11+ partner1",
```

Comment:
This is the longest platform name in the UI by quite some margin (so will reduce width for job results) - is there a shorter name that we could use? What are these builds? The name doesn't mean anything to me at the moment.

Thanks :-)